packages/@okta/i18n/src/properties/login_zh_CN.properties

```diff
@@ -192,7 +192,7 @@ errors.E0000110 = \u60A8\u6240\u8BBF\u95EE\u7684\u94FE\u63A5\u5DF2\u8FC7\u671F\u
 errors.E0000111 = \u65E0\u6CD5\u4FEE\u6539 {0} \u5BF9\u8C61\uFF0C\u56E0\u4E3A\u5B83\u662F\u53EA\u8BFB\u7684\u3002
 errors.E0000112 = \u65E0\u6CD5\u66F4\u65B0\u6B64\u7528\u6237\uFF0C\u56E0\u4E3A\u4ED6\u4EEC\u4ECD\u5904\u4E8E\u6FC0\u6D3B\u8FC7\u7A0B\u4E2D\u3002\u8BF7\u5728\u51E0\u5206\u949F\u540E\u91CD\u8BD5\u3002
 # {0} is a server-side error response, oftentimes coming directly from a vendor.
-errors.E0000113 = {0}\u3002
+errors.E0000113 = {0}.
```

Comment:
same here, a `.` is not equivalent.

Reply:
Send vendor email for updates.
I remember we fixed it last month. Have to check if it is XTM issue.